pipelines/DRP.yaml

```diff
@@ -14,3 +14,19 @@ tasks:
       # that's what the (unusual) ci_hsc repo is set up to have.
       connections.photoCalibName: "jointcal"
       useGlobalExternalPhotoCalib: false
+  deblend:
+    class: lsst.pipe.tasks.deblendCoaddSourcesPipeline.DeblendCoaddSourcesMultiTask
+    # Significantly limit the number of sources to deblend, which saves time
+    # in both deblending and measurement
+    config:
+      # Only run on the first 10 parents with between 5-10 children
```

Comment:
I think I'd rephrase this so it doesn't state the number of parents and children. It's all too easy for the code to change and this comment to be wrong.